omnisafe/common/simmer_agent.py

```diff
@@ -97,24 +104,31 @@ def get_greedy_action(
         # compute the error
         current_error = safety_budget - observation
         # blur the error
```

Comment:
Which the mean of blurring the error?

Reply:
This blur operation is following the implementation of simmer RL in https://github.com/huawei-noah/HEBO/tree/master/SIMMER. I think blur the error can make the trainning process stable.

Reply:
get it